src/frontends/paddle/tests/test_models/gen_scripts/generate_silu.py

```diff
@@ -0,0 +1,40 @@
+#
+# silu paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle as pdpd
+import sys
+
+
+def silu(name: str, x, data_type):
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(
+            name='input_x', shape=x.shape, dtype=data_type)
+        out = pdpd.fluid.layers.silu(x=node_x, name='silu')
```

Comment:
since fluid will be removed soon, pls try another silu implementation, thanks

Reply:
Solved, use `paddle.nn.functional.silu` instead